package.json

```diff
@@ -126,7 +126,7 @@
     "@babel/preset-typescript": "^7.18.6",
     "@commitlint/cli": "^17.1.2",
     "@commitlint/config-conventional": "^17.1.0",
-    "@pangolindex/sdk": "5.1.0",
+    "@pangolindex/sdk": "5.1.0-rc.2",
```

Comment:
any specific reason to use `5.1.0-rc.2` instead of `5.1.0` ? lets use  `5.1.0` as its latest version

Reply:
I didn't see that Hedera's new contracts were already in the master (sdk 5.1.0), that's why I was using the rc version